### DISCUSSION TITLE
Network drive as a music directory in Windows 10?

### DISCUSSION THREAD

**Discussion by progman27** [MusicPlayerDaemon/MPD#1990] [Q&A]
How to make a network drive as a music directory in Windows 10? MPD server version 0.23.15 x64 Windows. Thank you.

**Comment by malkav30** [comment]
Hi, late to the party but as I struggled on this one:
- either use nfs, e.g. `music_directory "nfs://<my_server>/path/to/music"`
- or connect the network share as a new volume through windows explorer (that's what I use), e.g. `music_directory "Z;\\path\\to\\music"`

smb seems not enabled in the provided windows binaries, at least I don't get it to work.
